templates/taxi/index.html

```diff
@@ -10,4 +10,5 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+  <p> You have visited this page {{ num_visits }} time{% if num_visits != 1 %}s{% endif %}.</p>
```

Comment:
Use `pluralize` as mentioned [in the checklist](https://github.com/tiron-vadym/py-taxi-service-authentication/blob/develop/checklist.md#code-efficiency).